tests/golos.publication_rewards_tests.cpp

```diff
@@ -378,21 +399,50 @@ class reward_calcs_tester : public golos_tester {
                         }
                     }
                     for (auto& r : cur_rewards) {
-                        _state.balances[r.first].vestamount += get_converted_to_vesting(r.second);
+                        if (_state.balances[r.first].vestclosed) {
+                            _state.balances[_forum_name].paymentsamount += (r.second + close_acc_delta);
```

Comment:
Why do you use `close_acc_delta` in this case? As I understand, this value for comparison paymentsamount, not for changes in model.